model/riscv_extensions.sail

```diff
@@ -310,3 +310,96 @@ function clause hartSupports(Ext_Sv57) = config extensions.Sv57.supported : bool
 enum clause extension = Ext_Smcntrpmf
 mapping clause extensionName = Ext_Smcntrpmf <-> "smcntrpmf"
 function clause hartSupports(Ext_Smcntrpmf) = config extensions.Smcntrpmf.supported
+
+let extensions_ordered_for_isa_string = [
+  // First append the single letter extensions.
+    Ext_M,
+    Ext_A,
+    Ext_F,
+    Ext_D,
+    // Ext_G,
+    // Ext_Q,
+    Ext_C,
+    Ext_B,
+    // Ext_P,
+    Ext_V,
+    //Ext_H,
+  // S and U are not valid extensions in the device-tree, whereas H is.
```

Comment:
Perhaps we could have uniform indentation in this block?  The `CODE_STYLE.md` recommends a 2-space block-level indent.